tools/internal_ci/linux/grpc_xds_k8s_install_test_driver.sh

```diff
@@ -361,3 +361,26 @@ local_setup_test_driver() {
   mkdir -p "${TEST_XML_OUTPUT_DIR}"
 }
 
+#######################################
+# Tag and push the given Docker image
+# Globals:
+#   INITIATOR: the initiator of this test script
+# Arguments:
+#   The Docker image name
+#   The Docker image original tag name
+#   The Docker image new tag name
+# Outputs:
+#   Writes the output to stdout, stderr, files
+#######################################
+tag_and_push_docker_image() {
+  local image_name="$1"
+  local from_tag="$2"
+  local to_tag="$3"
+
+  if [[ ${INITIATOR} == "kokoro" ]]; then
```

Comment:
I think we should avoid tight coupling with kokoro in this method. just tagging and pushing should be perfectly fine.

Reply:
Removed.

I worried about experimental runs might create unwanted Docker images. This condition is used to protect un-approved code being executed in our interop tests.